great_expectations/data_context/data_context/abstract_data_context.py

```diff
@@ -547,6 +550,19 @@ def stores(self) -> dict:
     def expectations_store_name(self) -> Optional[str]:
         return self.variables.expectations_store_name
 
+    @public_api
+    @new_method_or_class(version="0.17.2")
+    @expectations_store_name.setter
+    def expectations_store_name(self, value: str) -> None:
+        """Set the name of the expectations store.
+
+        Args:
+            value: New value for the expectations store name.
+        """
+
+        self.variables.expectations_store_name = value
+        self._save_project_config()
```

Comment:
Does this also work with `self.variables.save_config()`

Reply:
It sure does, but I thought better to use the method on the data context vs the method on a dependency (e.g. if the dependency changes we only would have to change the implementation of _save_project_config() vs all the callers).

Reply:
this is a great change. I can't believe we were missing this for so long.